understanding/focus-not-obscured-enhanced.html

```diff
@@ -56,150 +56,109 @@
+                  <dt>何をすればよいか</dt>
+                  <dd>項目がキーボードフォーカスを取得したときに、その項目が完全に見えることを保証する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who can't use a mouse need to see what has keyboard focus.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>マウスを使用できない人々は、キーボードフォーカスがどこにあるかを確認する必要がある。</dd>
                   
                </dl>
             </section>
             <section id="intent">
-               <h2>Intent</h2>
-               <p>The intent of this Success Criterion is to ensure that the item receiving keyboard
-                  focus is always visible in the user's viewport. For sighted people who rely on a keyboard
-                  (or on a device that operates through the keyboard interface, such as a switch or
-                  voice input), knowing the current point of focus is critical. The component with focus
-                  signals the interaction point on the page. Where users cannot see the item with focus,
-                  they may not know how to proceed, or may even think the system has become unresponsive.
+               <h2>意図</h2>
+               <p>この達成基準の意図は、キーボードフォーカスを受け取る項目が、利用者のビューポート内で常に表示されることを保証することである。キーボード (又はスイッチ、音声入力など、キーボードインターフェースを通じて操作するデバイス) に依存している目の見える人にとって、現在のフォーカス地点を知ることは非常に重要である。フォーカスを持つコンポーネントは、ページ上のインタラクション地点を示す。利用者がフォーカスを持つ項目を見ることができない場合、利用者はどのように進めればよいのかわからない、又はシステムが応答しなくなったとさえ思うかもしれない。
                </p>
-               <p>Typical types of content that can overlap focused items are sticky footers, sticky
-                  headers, and non-modal dialogs. As a user tabs through the page, these layers of content
-                  can hide the item receiving focus, along with its focus indicator.
+               <p>フォーカスされた項目と重なる可能性のある代表的なコンテンツの種類は、スティッキーフッター、スティッキーヘッダー、及び非モーダルダイアログである。利用者がページ内をタブで移動すると、これらのコンテンツのレイヤーにより、フォーカスを受け取った項目がフォーカスインジケータと共に覆い隠す可能性がある。
                </p>
-               <p>A notification implemented as sticky content, such as a cookie banner, will fail this
-                  Success Criterion if it partially covers a component receiving focus. Ways of passing
-                  include making the banner modal so the user has to dismiss the banner before navigating
-                  through the page, or using <a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding">scroll padding</a> so the banner does not overlap other content. Notifications that do not require user
-                  action could also meet this criterion by closing on loss of focus.
+               <p>Cookie バナーのようなスティッキーコンテンツとして実装された通知は、フォーカスを受け取るコンポーネントを部分的に覆う場合、この達成基準に失敗する。合格する方法には、利用者がページ内をナビゲートする前にバナーを閉じる必要があるようにバナーをモーダルにする、又はバナーが他のコンテンツと重ならないように <a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding">scroll-padding</a> を使用する、といったものがある。利用者のアクションを必要としない通知も、フォーカスが失なわれるときに閉じることでこの基準を満たす可能性がある。
```

Comment:
訳としては問題ないと思いますが、達成基準 2.4.11 との対比として、ニュアンスを少し調整してみました。いかがでしょうか？

```suggestion
               <p>Cookie バナーのようなスティッキーコンテンツとして実装された通知は、フォーカスを受け取るコンポーネントを部分的にでも覆う場合、この達成基準に失敗する。合格する方法には、利用者がページ内をナビゲートする前にバナーを閉じる必要があるようにバナーをモーダルにする、又はバナーが他のコンテンツと重ならないように <a href="https://www.w3.org/TR/css-scroll-snap/#propdef-scroll-padding">scroll-padding</a> を使用する、といったものがある。利用者のアクションを必要としない通知も、フォーカスが失なわれるときに閉じることでこの基準を満たす可能性がある。
```